app/views/bulk_update/recommendations_uploads/edit.html.erb

```diff
@@ -7,6 +7,8 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop" >
     <%= register_form_with(model: @recommendations_upload_form, url: bulk_update_recommendations_uploads_path) do |f| %>
+      <%= hidden_field_tag(:recommendations_upload_id, params[:id]) %>
```

Comment:
I didn't use `f.hidden_field` here because that would nest the hidden field under `bulk_update_recommendations_upload_form` in the params, like so ```ruby
"bulk_update_recommendations_upload_form"=>#<ActionController::Parameters {"id"=>"18"} permitted: false>
```
This would not work for the parent method:

```ruby
def recommendations_upload
      @recommendations_upload ||= provider.recommendations_uploads.find(params[:id] || params[:recommendations_upload_id])
    end
```